wagon.py

```diff
@@ -454,7 +454,7 @@ def _make_virtualenv(virtualenv_dir=None):
     if not virtualenv_dir:
         virtualenv_dir = tempfile.mkdtemp()
     logger.debug('Creating Virtualenv %s...', virtualenv_dir)
-    _run([sys.executable, '-m', 'virtualenv', virtualenv_dir])
+    _run([sys.executable, '-m', 'venv', virtualenv_dir])
```

Comment:
actually I think just `venv.create(virtualenv_dir)` (with an `import venv`) is better, no need to subprocess, the stdlib venv explicitly gives us a python-side public api